Source/WebCore/Scripts/tests/DocumentSyncData.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2024 Apple Inc. All rights reserved.
+ * Copyright (C) 2025 Apple Inc. All rights reserved.
```

Comment:
I guess this needs to be 2024-2025

Reply:
Files generated by the script before were 2024. Files generated by the script now are 2025... right?